synapse/config/repository.py

```diff
@@ -202,6 +215,21 @@ def generate_config_section(self, data_dir_path, **kwargs):
 
         return (
             r"""
```

Comment:
All this stuff is docced at https://github.com/matrix-org/synapse/blob/develop/docs/code_style.rst#configuration-file-format btw